### PR TITLE
Fix bowtie2 options in ATAC

### PR DESCRIPTION
**Pull request by lldelisle**
I am really sorry for this, I don't understand how it happened...

### CODE REVIEW

File: workflows/epigenetics/atacseq/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [1.0] 2024-11-28
+
+### Manual update
+
+Use the options like written in the README: in all previous versions, bowtie2 was running with `--very-sensitive` option but without allowing dovetail and without fragment length up to 1kb. Bowtie2 has been adjusted and is now running with these options: allow dovetail and fragment length up to 1kb.
+
+I am really sorry for this.
+
```
Comment (mvdbeek) [inline workflows/epigenetics/atacseq/CHANGELOG.md:10]:
```suggestion
```

File: workflows/epigenetics/atacseq/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [1.0] 2024-11-28
+
+### Manual update
+
+Use the options like written in the README: in all previous versions, bowtie2 was running with `--very-sensitive` option but without allowing dovetail and without fragment length up to 1kb. Bowtie2 has been adjusted and is now running with these options: allow dovetail and fragment length up to 1kb.
```
Comment (mvdbeek) [inline workflows/epigenetics/atacseq/CHANGELOG.md:7]:
```suggestion
Use the options like written in the README: in all previous versions, bowtie2 was running with `--very-sensitive` option but without allowing dovetail and without fragment length up to 1kb. Bowtie2 has been adjusted and is now running with these options: allow dovetail and fragment length up to 1kb.

```
😆 sorry about the missing newline here.

**Comment by mvdbeek** [comment]
> I don't understand how it happened...

My theory is that paired_options_selector was a boolean conditional test parameter, that was switched to a select in a tool upgrade in order to pacify an overly strict linting rule, and the default was then used in a workflow upgrade.

**Review by mvdbeek — APPROVED** [review]
No need to worry, this is why we have versioned workflows. Thanks for following up :)

**Comment by lldelisle** [comment]
Yes probably, my original version was bowtie2 2.4.2 and when I put it into iwc I put 2.4.5+galaxy1...

**Comment by lldelisle** [comment]
This reminds me why it is important to have these options:
Expected text '30410 (10.82%) aligned concordantly 0 times' in output ('280964 reads; of these:
  280964 (100.00%) were paired; of these:
    7282 (2.59%) aligned concordantly 0 times...

**Review by mvdbeek — APPROVED** [review]

**Pull request merged**